src/main/scala/fundamentals/level03/ExceptionExercises.scala

```diff
@@ -40,10 +40,7 @@ object ExceptionExercises {
     * scala> getName("")
     * = EmptyNameException: provided name is empty
     *
-    * scala> getName("   ")
-    * = EmptyNameException: provided name is empty
-    *
-    * Hint: use the trim and isEmpty methods on String
+    * Hint: use the isEmpty method on String
```

Comment:
You're going to have to update getName in Exceptions2EitherExercises as well.

Reply:
Oops! I'm starting to like this whole PR process.